src/test/build-all-projects.ts

```diff
@@ -27,49 +29,55 @@ describe('Setup test environment for Yo Office build tests', () => {
-              }); 
-            });
+        if (parsedProjectJsonData.projectTypes[projectTemplates[j]].typescript)
+        {
+            describe('Install and build projects', () => {
+                let projectType = projectTemplates[j];        
+                let host = hostsTemplates[i];
+                let scriptType = typescript;
+                let projectName = projectType + host + typescript;
+                let projectFolder = path.join(__dirname, '/', projectName);          
+            
+                describe(stringBuildStart +  host + space + projectType + space + typescript, () => {
+                    it(stringBuildSucceeds,function(done){  
+                        _generateProject(projectType, projectName, host, projectFolder, scriptType);
+                        _buildProject(projectFolder, projectType);
+                        done();                    
+                      });
+                  }); 
+                });
+            }
         }
     }
-
+    
 // Build Javascript project types for all supported hosts
 for (var i = 0; i < hostsTemplates.length; i++)
 {
-    for (var j = 0; j < jsTemplates.length; j++)
+    for (var j = 0; j < projectTemplates.length; j++)
     {
-        describe('Install and build projects', () => {
-            let projectType = jsTemplates[j];        
-            let host = hostsTemplates[i];
-            let scriptType = javascript;
-            let projectName = projectType + host + javascript;
-            let projectFolder = path.join(__dirname, '/', projectName);  
-        
-            describe(stringBuildStart +  host + space + projectType + space + javascript, () => {
-                it(stringBuildSucceeds,function(done){         
-                    _generateProject(projectType, projectName, host, projectFolder, scriptType);
-                    _buildProject(projectFolder, projectType);
-                    done();
-                  });
-              }); 
-            });
+        if (parsedProjectJsonData.projectTypes[projectTemplates[j]].javascript)
+        {
+            describe('Install and build projects', () => {
+                let projectType = projectTemplates[j];        
+                let host = hostsTemplates[i];
+                let scriptType = javascript;
+                let projectName = projectType + host + javascript;
+                let projectFolder = path.join(__dirname, '/', projectName);          
+            
+                describe(stringBuildStart +  host + space + projectType + space + typescript, () => {
+                    it(stringBuildSucceeds,function(done){  
+                        _generateProject(projectType, projectName, host, projectFolder, scriptType);
+                        _buildProject(projectFolder, projectType);
+                        done();                    
+                      });
+                  }); 
+                });
+            }
         }
-    }    
-    
+    }
```

Comment:
>} [](start = 4, length = 1)

nit: I think that the indentation is not correct (this happens as well in the **for** above)